src/pages/documentation/api-changelog.md

```diff
@@ -10,36 +10,36 @@ For more information, see [Mobile Core API reference](./mobile-core/api-referenc
-| [setPrivacyStatus](./privacy-and-gdpr.md#set-and-get-privacy-status) | setPrivacyStatus: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/config-ios/sdk-methods.html?lang=en) | [Android](https://experienceleague.adobe.com/docs/mobile-services/android/configuration-android/methods.html?lang=en)) |
-| [setLogLevel](getting-started/track-events.md#enable-debug-logging) | setDebugLogging: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/config-ios/sdk-methods.html?lang=en) | [Android](https://experienceleague.adobe.com/docs/mobile-services/android/configuration-android/methods.html?lang=en)) |
-| [configureWithFileInPath:](./api-changelog.md) | overrideConfigPath: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/config-ios/sdk-methods.html?lang=en) | [Android](https://experienceleague.adobe.com/docs/mobile-services/android/configuration-android/methods.html?lang=en)) |
-| [configureWithAppId:](./api-changelog.md) | Not applicable |
-| [updateConfiguration:](./api-changelog.md) | Not applicable |
-| setAppGroup: | setAppGroup: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/ios-ext/ios-ext.html?lang=en)) |
-| [trackState:data:](./mobile-core/api-reference.md#track-app-states-and-views) | trackState:data: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/analytics-ios/states.html?lang=en) | [Android](https://experienceleague.adobe.com/docs/mobile-services/android/analytics-android/states.html?lang=en)) |
-| [trackAction:data:](./mobile-core/api-reference.md#track-app-states-and-views) | trackAction:data: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/analytics-ios/actions.html?lang=en) | [Android](https://experienceleague.adobe.com/docs/mobile-services/android/analytics-android/actions.html?lang=en)) |
-| [collectPII:](./mobile-core/api-reference.md#collect-pii) | collectPII: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/analytics-ios/postbacks/c-pii-postbacks.html?lang=en) | [Android](https://experienceleague.adobe.com/docs/mobile-services/android/analytics-android/postbacks/c-pii-postbacks.html?lang=en)) |
-| [getSdkIdentities:](./mobile-core/identity/api-reference.md#getidentifiers) | getAllIdentifiersAsync: ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/privacy-gdpr-ios/c-mob-gdpr-ret-stored-ids-ios.html?lang=en) | [Android](https://experienceleague.adobe.com/docs/mobile-services/android/gdpr-privacy-android/c-mob-gdpr-ret-stored-ids-android.html?lang=en)) |
+| [extensionVersion:](./mobile-core/configuration/api-reference.md#extensionversion) | version/getVersion ([iOS](https://experienceleague.adobe.com/docs/mobile-services/ios/config-ios/sdk-methods.html?lang=en) \| [Android](https://experienceleague.adobe.com/docs/mobile-services/android/configuration-android/methods.html?lang=en)) |
```

Comment:
I think we can skip this link since this API is per extension not just for core

Reply:
@emdobrin But it is still section for MobileCore. Every extension covers extensionVersion method right?

Reply:
Yup. My thinking was that if you compare the 4x SDK getVersion with MobileCore extensionVersion they are not equivalent, because you may have multiple extensions